magprime/templates/regextra.html

```diff
@@ -9,9 +9,14 @@
         }
         $("input:radio[name='amount_extra'][value=40]").parents('.btn').append($('#guitar_keychain'));
         $('#guitar_keychain').on('click', function(e) { e.stopPropagation(); })
-        if ($("input:radio[name='amount_extra'][value=250]")) {
+        if ($("input:radio[name='amount_extra'][value={{ c.SUPPORTER_LEVEL }}]")) {
```

Comment:
Oh duh, that was quite an oversight on my part. I'll leave this tab open until I get around to cleaning it up.